lib/new_relic/agent/instrumentation/rails_notifications/action_controller.rb

```diff
@@ -38,6 +38,9 @@
       halted_callback
       unpermitted_parameters]
 
+    # TODO: Add send_steam to subs list when Rails 7.2 becomes avaliable
+    subs << 'send_stream' if Rails::VERSION::MAJOR.to_i >= 7 && Rails::VERSION::MINOR.to_i >= 2
```

Comment:
I suggest going ahead and satisfying the TODO right now, and only having the conditional check live in the tests where you have it.

The ever-so-slightly-more complex regex pattern that results from "or"-ing (`|`) in the new `'send_stream'` string will make for negligible performance impact on Rails < 7.2 users while they wait for 7.2 to be released.

And then when 7.2 drops, they can automatically benefit from the new subscription type without needing to upgrade their APM agent.

Reply:
Thank you! I like this suggestion and have updated ◡̈